pages/WatchList.js

```
@@ -1,94 +1,66 @@
+      headerSearchBarOptions: showWatchList
+        ? {
+            placeholder: i18n.t('search'),
+            hideWhenScrolling: false,
+            onChangeText: (e) => handleSearch(e.nativeEvent.text ?? ''),
+            onCancelButtonPress: () => {
+              setSearchQuery('');
+            },
+          }
+        : undefined,
+    });
+  }, [navigation, showWatchList, handleSearch]);
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)